integration-cli/docker_api_network_test.go

```diff
@@ -172,6 +174,54 @@ func (s *DockerSuite) TestAPINetworkConnectDisconnect(c *check.C) {
 	// delete the network
 	deleteNetwork(c, nr.ID, true)
 }
+func delInterface(c *check.C, ifName string) {
+	icmd.RunCommand("ip", "link", "delete", ifName).Assert(c, icmd.Success)
+	icmd.RunCommand("iptables", "-t", "nat", "--flush").Assert(c, icmd.Success)
+	icmd.RunCommand("iptables", "--flush").Assert(c, icmd.Success)
+}
+func (s *DockerDaemonSuite) TestAPIDaemonNetworkPools(c *check.C) {
+	testRequires(c, DaemonIsLinux, SameHostDaemon)
+
+	// Remove docker0 bridge and the start daemon defining the predefined address pools
+	defaultNetworkBridge := "docker0"
+	delInterface(c, defaultNetworkBridge)
+	s.d.Start(c,
+		"--default-node-local-address-pools", "base=175.30.0.0/16,size=16",
+		"--default-node-local-address-pools", "base=175.33.0.0/16,size=24")
+
+	// Verify bridge network's subnet
+	cli, err := s.d.NewClient()
+	c.Assert(err, checker.IsNil)
+	defer cli.Close()
+	out, err := cli.NetworkInspect(context.Background(), "bridge", types.NetworkInspectOptions{})
+	c.Assert(err, check.IsNil)
+	//c.Assert(out, checker.Contains, "175.30.0.0/16")
+	c.Assert(out.IPAM.Config[0].Subnet, checker.Equals, "175.30.0.0/16")
+
+	// Create a bridge network and verify its subnet is the second default pool
```

Comment:
It looks like these are separate cases? Can you split this into separate `Test` functions?

Reply:
It is same test case. We are testing it by creating 2 different network and make sure they fall into same user defined subnet. thats all.